tests/device_tests/bitcoin/test_descriptors.py

```diff
@@ -160,7 +169,22 @@ def test_descriptors(client: Client, coin, account, purpose, script_type, descri
         if client.features.model != "1":
             IF = InputFlowShowXpubQRCode(client)
             client.set_input_flow(IF.get())
-        res = btc._get_descriptor(
+        res = btc_cli._get_descriptor(
             client, coin, account, purpose, script_type, show_display=True
         )
         assert res == descriptors
+
+        if client.features.model != "1":
+            IF = InputFlowShowXpubQRCode(client)
+            client.set_input_flow(IF.get())
+        address_n = _address_n(purpose, coin, account, script_type)
+        res = btc.get_public_node(
+            client,
+            _address_n(purpose, coin, account, script_type),
+            show_display=True,
+            coin_name=coin,
+            script_type=script_type,
+            ignore_xpub_magic=True,
+            unlock_path=btc_cli.get_unlock_path(address_n),
+        )
+        assert res.descriptor == descriptors
```

Comment:
this whole block should be under the model check, no?

also please make a separate test for this

(also please start using `client.model is models.T1B1` instead of string comparison from now on)

Reply:
fixed in [e4f6b6b](https://github.com/trezor/trezor-firmware/pull/3545/commits/e4f6b6b6343455f445abfb021533c8ae39f66f92)
will need to update T1 fixtures still